openmrs-compose.yaml

```diff
@@ -0,0 +1,42 @@
+version: '2.1'
+services:
+  mysql:
+    image: mysql:5.7
+    command: "mysqld --character-set-server=utf8 --collation-server=utf8_general_ci"
+    restart: always
+    ports:
+      - "3306:3306"
+    container_name: openmrs-fhir-mysql
+    healthcheck:
+      test: "exit 0"
+    environment:
+      - MYSQL_DATABASE=openmrs
+      - MYSQL_ROOT_PASSWORD=debezium
+      - MYSQL_USER=mysqluser
+      - MYSQL_PASSWORD=mysqlpw
+      - MYSQL_ROOT_HOST=%    # Allow docker containers to connect to mysql
+    volumes:
+      - ./utils/mysql.cnf:/etc/mysql/conf.d/custom.cnf  # mysql config preconfigured to allow binlog/debezium
+      - ./data:/var/lib/mysql
```

Comment:
What goes into the `./data` directory? I see that you have excluded it in `.gitignore`, is it supposed to be an empty directory? If yes, can't we just say `- /var/lib/mysql` here? If not and `./data` has some necessary content, why not have it under git?

Reply:
this should be fixed now, having removed ./data